src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.ts

```diff
@@ -6,20 +6,29 @@ import { Component, Input, OnChanges } from '@angular/core';
   styleUrls: ['./usage-bar.component.scss']
 })
 export class UsageBarComponent implements OnChanges {
+  @Input()
+  totalAmount: number;
+  @Input()
+  usedAmount: number;
   @Input()
   totalBytes: number;
   @Input()
   usedBytes: number;
 
+  freeAmount: number;
+  freeBytes: number;
   usedPercentage: number;
   freePercentage: number;
-  freeBytes: number;
 
   constructor() {}
 
   ngOnChanges() {
-    this.usedPercentage = Math.round((this.usedBytes / this.totalBytes) * 100);
-    this.freePercentage = 100 - this.usedPercentage;
+    this.freeAmount = this.totalAmount - this.usedAmount;
     this.freeBytes = this.totalBytes - this.usedBytes;
+    this.usedPercentage =
+      this.totalAmount > 0
+        ? Math.round((this.usedAmount / this.totalAmount) * 100)
+        : Math.round((this.usedBytes / this.totalBytes) * 100);
+    this.freePercentage = 100 - this.usedPercentage;
```

Comment:
If `isBinary` is used:
```suggestion
    this.usedPercentage = this.total > 0 ? Math.round((this.used / this.total) * 100) : 100; // total should not be 0 but if it is 100 percent is used as we don't have any capacity (no matter for what)
    this.freePercentage = 100 - this.usedPercentage;
```

Reply:
Hey! I found a problem with usedPercantange value in case of total=0, which is here set to 100. From the img attached it can be seen that even if we have used and total equal to 0, the usage is shown to be 100%, which can be misleading . I think we should show 0% instead 
![Screenshot from 2020-06-25 10-51-29](https://user-images.githubusercontent.com/23611106/85659203-e71bcb80-b6d1-11ea-9399-7ef220fc696e.png)